recipe/meta.yaml

```
@@ -8,17 +8,18 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pynng-{{ version }}.tar.gz
   sha256: 5edd3ccbeda2264d750d81533a11d45b5efbf8d99da65ba1f4569636f0eb9b70
+  patches:
+    - fix-windows-builds.patch
 
 build:
-  number: 2
+  number: 3
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [not linux]
 
 requirements:
   build:
     - cmake
     - {{ compiler('c') }}
-    - make
+    - ninja
```

```suggestion
    - ninja
    - make  # [unix]
```
Please keep `make` in the dependencies at least for unix